<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>485</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Easy App Registrar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/res/img/APP.png</normaloff>:/res/img/APP.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>app-name</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>exec-filepath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>0</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load defaults</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/out.svg</normaloff>:/res/img/out.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/start.svg</normaloff>:/res/img/start.svg</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>icon-filepath</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>71</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>App name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>91</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>App filepath</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>91</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Icon filepath</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save resources</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/guard.svg</normaloff>:/res/img/guard.svg</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Global installation</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/pc.svg</normaloff>:/res/img/pc.svg</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>User installation</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/user.svg</normaloff>:/res/img/user.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>761</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Launcher name: &lt;empty&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>761</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder name: &lt;empty&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as default</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/in.svg</normaloff>:/res/img/in.svg</iconset>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>271</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit mode</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/edit.svg</normaloff>:/res/img/edit.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add more</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/folder.svg</normaloff>:/res/img/folder.svg</iconset>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/folder.svg</normaloff>:/res/img/folder.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/res/img/folder.svg</normaloff>:/res/img/folder.svg</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>118</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionReset_2"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/settings.svg</normaloff>:/res/img/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionReset_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/trash.svg</normaloff>:/res/img/trash.svg</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
